build.gradle

```diff
@@ -0,0 +1,44 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
+
+repositories {
+    mavenLocal()
+    maven {
+        url = uri('http://repo.maven.apache.org/maven2')
+    }
+}
+
+dependencies {
+    implementation 'org.seleniumhq.selenium:selenium-java:3.141.59'
+    implementation 'com.aventstack:extentreports:3.1.5'
+    implementation 'org.apache.poi:poi:3.9'
+    implementation 'org.apache.poi:poi-ooxml:3.9'
+    implementation 'com.google.inject:guice:4.2.2'
+    implementation 'net.jmob:guice.conf:1.4.3'
+    testImplementation 'org.junit.platform:junit-platform-launcher:1.7.0'
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.7.0'
+    testImplementation 'org.testng:testng:7.3.0'
+    compileOnly 'org.projectlombok:lombok:1.18.16'
```

Comment:
Mógłbyś podać przykład? Powiem szczerze, że od początku nauki go używam i nie miałem żandych problemów.

Reply:
That's most likely because you've barely used it, and did not had to debug your code much, or was able to use IntelliJ which has quite good support for lombok.

Main issue I have with lombok is that it generates methods/constructors during build, and you often can't trace method calls while looking for a root cause of an issue.

Another thing would be with access modifiers, while in tests it usually does not matter much, but when you are building a library that's meant to be shared with other teams/projects/outside of your organisation access modifiers allow you to differentiate internal classes/methods from ones that should be used by your peers. I don't trust something I can't see, and because of how lombok works, I'm never quite sure it did access modifiers the way I intended...